src/components/exercises/ExerciseItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
```

Comment:
I was under the impression that you had decided to ditch the TypeScript. But this is a new file and it feels like you chose to use TypeScript here.

Reply:
We have chosen to keeo TypeScript for the MVP for now because we don't have the time to delete it everywhere cleanly.